.github/workflows/build.yaml

```
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4]
+        php: [7.4]
         coverage: ["true"]
         include:
         - php: 8.0
```

If we're dropping the support for PHP < 7.4 ⬇️ 

```suggestion
        php: [7.4, 8.0, 8.1, 8.2]
        coverage: ["true"]
        include:
        - php: 8.2
```

Otherwise ⬇️ 
```suggestion
        php: [7.2, 7.3, 7.4, 8.0, 8.1, 8.2]
        coverage: ["true"]
        include:
        - php: 8.2
```

In any case, all the following stable PHP versions should probably be used too.